composer.json

```diff
@@ -30,7 +30,7 @@
         "laminas/laminas-stdlib": "^3.2.1"
     },
     "require-dev": {
-        "laminas/laminas-coding-standard": "~1.0.0",
+        "laminas/laminas-coding-standard": "^2.0",
```

Comment:
Please use `~2.0.0`. New minor versions may introduce new sniffs... which would make CS checks fail. By using the `~` constraint, we can opt-in to those new sniffs when we are ready, instead of having to suddenly update for a new patch.

Reply:
👍🏻 Thanks for the explanation. Done.